### PR TITLE
Add new JEP: GSoC Budgeting and Expenses

### PR DESCRIPTION
**Pull request by oleg-nenashev**
CC @martinda and @christ66 as GSoC Org admins and also @rtyler as SPI board member and active Jenkins contributor

### CODE REVIEW

File: jep/0000/README.adoc [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+== Abstract
+
+This document documents the Google Summer of Code (GSoC) budgeting process.
+
+== Specification
+
+=== Budget sources
+
+The budget is formed from...
```
Comment (martinda) [inline jep/0000/README.adoc]: The budget revenue sources are:

File: jep/0000/README.adoc [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+open-source conferences like FOSDEM, etc.
+** Travel grants may cover: hotel, transportation costs, visa fees, travel insurance, meals
+** Transportation costs may be covered only for economy-class travel
+** Travel grants do NOT cover: conference fees, alcohol, souvenirs, etc.
+2. GSoC Mentor Summit travel grants for GSoC mentors and org admins
+** The budget is provided according to the rules defined in link:https://developers.google.com/open-source/gsoc/help/org-payments[GSoC Org Payments] (as of 2018 - 2200 USD)
+** Mentor travel grants may cover: transportation costs, visa fees, meals
+** Transportation costs may be covered only for economy-class travel
+** Travel grants do NOT cover travel expenses to any other events
+** Travel grants do NOT cover the hotel costs as long as it covered by the event organizer
```
Comment (martinda) [inline jep/0000/README.adoc]: How about: Travel grants do NOT cover expenses that are already covered event organizers, or other types of sponsors.

File: jep/0000/README.adoc [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+** Project expenses should happen within the GSoC timeframe between start of Community Bonding and end of the last coding phase
+** GSoC budget may not be used to cover generic equipment costs: laptop, IDEs, internet, etc.
+** GSoC budget may not be used to cover software license costs.
+The most of the projects are expected to be doable with only open-source or freeware software,
+otherwise it is up to the students and mentors to get the setup.
+** Total project expenses may not exceed the amount being payed by Google per project (as of 2018 - 500 USD)
+4. Promotional materials
+** GSoC budget may be used to promote Google Summer of Code by distributing swag, e.g. stickers, T-shirts and other items.
+Swag may be sent to participants or distributed at community events.
+** GSoC budget may be used to cover swag production costs and delivery costs by common mail
```
Comment (martinda) [inline jep/0000/README.adoc]: common mail -> regular mail?

File: jep/0000/README.adoc [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+** Swag may be sent out to active GSoC participants, including GSoC students, mentors, org admins, applicants, and other contributors who participated in GSoC
+** GSoC budget may not be used for any kind of payed advertisement or
+payments to external individuals/organizations for promotion.
+** GSoC budget may not be used to send out promotional materials which explicitly or implicitly reference other trademarks or commercial companies
+
+=== Approval process
+
+* All expenses should be pre-approved with
+Jenkins GSoC org admins before the expenses are incurred, and as a pre-condition for paymen
```
Comment (martinda) [inline jep/0000/README.adoc]: paymen -> payment

File: jep/0000/README.adoc [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+=== Payment process
+
+* Jenkins project does not pay for expenses,
+but it reimburses expenses post-factum to students, mentors or Org Admins
+* Reimbursements are performed according to the SPI process defined
+link:https://wiki.jenkins.io/display/JENKINS/Reimbursement+of+Expenses+via+SPI[Reimbursement of Expenses via SPI]
+* It may take a while to get an expense report reimbursed
+(see the _Liability_ section)
+* It is **recommended** that GSoC mentors do the payments for students
+and then reimburse the expenses
```
Comment (martinda) [inline jep/0000/README.adoc]: then seek reimbursement for the expenses. Pre-approval is still required in this case.

File: jep/0000/README.adoc [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+
+=== Liability
+
+Jenkins project has no legal entity which can commit
+on expenses.
+As GSoC organization, we also
```
Comment (martinda) [inline jep/0000/README.adoc]: "we also" does not end the sentence properly.

**Comment by bitwiseman** [comment]
@kohsuke Will be reviewer for this JEP this is ready for draft.

**Comment by oleg-nenashev** [comment]
Thanks @bitwiseman! sorry for not getting to comments immediately

**Pull request merged**